Documentation/INTERNALS.md

```diff
@@ -187,28 +212,26 @@ Here is the actual first part of the generated code for `message Foo` above:
 
 ```swift
 public struct Foo: ProtobufGeneratedMessage {
-  public var protoMessageName: String {return "Foo"}
-  public var protoPackageName: String {return ""}
-  public var jsonFieldNames: [String: Int] {return [
-    "fooField": 1,
-  ]}
-  public var protoFieldNames: [String: Int] {return [
-    "fooField": 1,
-  ]}
+  static let protoMessageName: String = "Foo"
```

Comment:
Does it make sense to just put prose in with references to the generated properties/functions and not try to repeat the generated code since it will surely be stale quickly?